wasm/vm_num_test.go

```diff
@@ -1,3 +1,44 @@
 package wasm
 
-// fixme
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNumOps(t *testing.T) {
```

Comment:
thank you for trying grouping all instruction together but I don't think this is the way to go because there's several types of num operations (https://webassembly.github.io/spec/core/syntax/instructions.html#numeric-instructions).

I want separated unit tests for each instruction and each is written in table driven manner